addrmgr/error.go

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) 2021 The Decred developers
+// Use of this source code is governed by an ISC
+// license that can be found in the LICENSE file.
+
+package addrmgr
+
+// ErrorKind identifies a kind of error.  It has full support for errors.Is
+// and errors.As, so the caller can directly check against an error kind
+// when determining the reason for an error.
+type ErrorKind string
+
+// These constants are used to identify a specific RuleError.
+const (
+	// ErrAddressNotFound indicates that an operation in the address manager
+	// failed due to encountering an unknown address.
```

Comment:
I don't think `ErrAddressNotFound` is a failure related to encountered an unknown address as the comment states, it's an address lookup failure.

Reply:
Updated